plugin/src/test/java/org/opensearch/ml/rest/MLCommonsRestTestCase.java

```diff
@@ -746,6 +748,47 @@ public String registerModel(String input) throws IOException {
         return parseTaskIdFromResponse(response);
     }
 
+    public void registerMLAgent(RestClient client, String input, Consumer<Map<String, Object>> function) throws IOException {
```

Comment:
Just curious why don't we add `@Test` with the test methods here?

Reply:
not needed once you extended OpenSearchRestTestCase.